src/components/signup/SmsForm.js

```diff
@@ -202,16 +202,15 @@ const SMSAction = ({
         if (!value) {
           value = smsRateLimitNormalized
         }
-        setWaitTime(value)
+        setWaitTime(parseInt(value))
```

Comment:
1. i suppose. you could do it once at line 187. also wrap it ot useMemo to avoid executing division each render. Because division by 10 requires floating point operations while division by 2 is much faster as ti's simply translated to the `shr` CPU bit instructions
2. aslo pls review config.js. the most correct solution is to put parseInt / parseFloat here. Also pls re check all config options, maybe there're some other numbers which could be passed as the strings

Reply:
Actually, this parseInt() is not needed, only in setTimeout()

Reply:
okay, please check also config.js and push the fixes if any